src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4176,40 +4176,61 @@ .FORWARDHELPCATEGORY Cmdlet
     }
     elseif ($help -ne $null)
     {
-        $customPagerCommand = """"
-        $customPagerCommandArgs = """"
-        $customPagerCommandLine = """"
-
-        if ($env:PAGER)
-        {
-            $customPagerCommandLine = $env:PAGER
+        # By default use more on Windows and less on Linux.
+        if ($IsWindows) {
```

Comment:
```suggestion
        if ($IsWindows)
        {
```

Reply:
@RDIL It is PowerShell code and we should use PowerShell formatting style.

Reply:
I'm aware
https://github.com/PowerShell/PowerShell/pull/9033#issuecomment-469343958